doc/documentation.rst

```diff
@@ -48,22 +48,22 @@ Documentation for private libraries may also be built with:
 
   $ dune build @doc-private
 
-But these libraries will not be in the main html listing above, since they do not
-belong to any particular package. But the generated html will still be found in
+But these libraries will not be in the main ``_html`` listing above, since they don't
```

Comment:
Note that in the original paragraph, html does not refer to the path `_html`, but html as in "the generated documentation written in html markup". So we should say revert to generated HTML, or even consider moving away from this phrasing altogether. If it confused you, I'm sure others are scratching their head as well.

Reply:
Agreed! This should be HTML. I'll update it.